libs/cardano-ledger-test/cardano-ledger-test.cabal

```
@@ -143,7 +143,7 @@ library
     cardano-ledger-mary,
     cardano-ledger-shelley:{cardano-ledger-shelley, testlib},
     cardano-ledger-shelley-ma-test,
-    cardano-ledger-shelley-test,
+    cardano-ledger-shelley-test >=1.6,
```

Same here. `cardano-ledger-test` can't use any other version except the one in this repo.
```suggestion
    cardano-ledger-shelley-test,
```